src/kernel/arch/x86/arch.zig

```diff
@@ -525,7 +525,7 @@ pub fn initMem(mb_info: BootPayload) Allocator.Error!MemProfile {
 /// Error: std.mem.Allocator.Error
 ///     OutOfMemory - There wasn't enough memory to allocate what was needed
 ///
-pub fn initKeyboard(allocator: Allocator) Allocator.Error!*Keyboard {
+pub fn initKeyboard(allocator: Allocator) Allocator.Error!*keyboard.Keyboard {
```

Comment:
We should still be able to just use `Keyboard` without the `keyboard.` prefix.

Reply:
Ah that was an accident, didn't notice that was in the commit.